### DISCUSSION TITLE
Color Previews for Cmp Menu Like VsCode

### DISCUSSION THREAD

**Discussion by EazyAhmed** [hrsh7th/nvim-cmp#1651] [General]
I am coding html and css. In vscode in the completion menu any item that has a hex Color value shows a box next to it with a preview of what that color looks like. is there a way to acheive this with nvim cmp? 
[(https://i.stack.imgur.com/HXSIS.png)](url)
like this

**Comment by Shougo** [comment]
The linked URI seems wrong.

https://i.stack.imgur.com/HXSIS.png

**Reply by EazyAhmed** [reply]
https://github.com/brenoprata10/nvim-highlight-colors
Nvm now i found a plugin that does just that.
